metricbeat/module/docker/container/data.go

```diff
@@ -17,24 +16,45 @@ func eventsMapping(containersList []dc.APIContainers) []common.MapStr {
 	}
 	return myEvents
 }
-func eventMapping(mycontainer *dc.APIContainers) common.MapStr {
+
+func eventMapping(cont *dc.APIContainers) common.MapStr {
 
 	event := common.MapStr{
-		"@timestamp": time.Now(),
+		"timestamp": time.Now(),
```

Comment:
Not sure I understand this, does it mean we'll have both `@timestamp` and `timestamp`? Seems unnecessary.